internal/pipeline/cmd.go

```diff
@@ -53,11 +53,15 @@ func (r *Runner) preRunE(c *cobra.Command, args []string) error {
 }
 
 func (r *Runner) runE(c *cobra.Command, args []string) error {
+	err := cmdutil.DockerCmdAvailable()
+	if err != nil {
+		return err
+	}
```

Comment:
I am wondering what exactly does a user see when they encounter this error ? Can you share the exact output.

Reply:
ran `kpt pipeline run` from my terminal while the docker daemon was not running and received the following:
```
error: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
docker is required to run this command. 
To install docker, follow the instructions at https://docs.docker.com/get-docker/
```